### PR TITLE
version upgrade to 6.0.3.5

### PR DESCRIPTION
**Pull request by olehyev138**
version upgrade from 6.0.3.2 to 6.0.3.5

### CODE REVIEW

File: app/controllers/accounts_controller.rb [hunk 1]
```diff
@@ -40,24 +40,33 @@ def show # rubocop:todo Metrics/CyclomaticComplexity
   end
 
   def create
-    @account = Accounts::Initialize.call(
```
Comment (zhinfuhnya) [inline app/controllers/accounts_controller.rb:43]: @olehyev138 are you sure we should roll back reCaptcha changes?

https://github.com/CoMakery/comakery-server/pull/1950

**Comment by aquabu** [comment]
@olehyev138 please check if the branch needs to be updated. There have been a lot of recent changes on `acceptance`

File: app/controllers/accounts_controller.rb (hunk 1 above)
Comment (olehyev138) [reply app/controllers/accounts_controller.rb:43]: I made the branch a few days ago and so never updated some data from origin branch

Comment (olehyev138) [reply app/controllers/accounts_controller.rb:43]: I mainly focus to upgrade Rails version

**Pull request closed without merging**